src/components/uri/Visualization.tsx

```diff
@@ -388,9 +388,13 @@ export class VisualizationWrapped extends React.Component<
                 );
             }
             case VisualizationTypes.PUSHPIN:
+                const processedVisualizationObject = {
+                    ...mdObject,
+                    content: fillMissingTitles(mdObject.content, locale),
```

Comment:
Is this change an improvement?

Reply:
this change fixes[issue](https://jira.intgdc.com/browse/SD-868?focusedCommentId=1265835&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1265835)